frontend/src/libs/packages/api/http-api.package.ts

```diff
@@ -105,6 +105,11 @@ class HttpApi implements IHttpApi {
       }),
     )) as ServerErrorResponse;
 
+    if (response.status === HttpCode.UNAUTHORIZED) {
```

Comment:
We may also make it more correct way - like applying `jwt-decode` package to retrieve exp_time and check it.
Because I thought a little bit - and with this approach any user action and fires UNAUTHORIZED - will trigger logout

Reply:
As alternative to decoding token and checking it's exp_time -- we may just check additional response error message if it's equal to `Invalid Token`